Cargo.toml

```diff
@@ -90,3 +90,12 @@ codegen-units = 1
 [patch.crates-io]
 # TODO: Remove once chacha20poly1305 0.10 appears in libp2p's dependencies
 chacha20poly1305 = { git = "https://github.com/RustCrypto/AEADs", rev = "06dbfb5571687fd1bbe9d3c9b2193a1ba17f8e99" }
+
+[patch."https://github.com/paritytech/substrate.git"]
```

Comment:
Please add comment as to why this is necessary because it is not very obvious and might break when set of dependencies change in snowfork (we might have to add or be able to remove some of the entries here). See above example, it explains why it is necessary (kind of) and when we can get rid of it.

Reply:
Okay. Added it.